crates/swc/tests/fixture/issues-4xxx/4953/output/index.js

```diff
@@ -2,6 +2,7 @@
 function _default(e, t) {
     "use strict";
     t.vr = function(e, t, r) {
+        try {} catch (e) {}
```

Comment:
`dead_code` is false.

https://github.com/swc-project/swc/blob/1388160a117e8768a22f243f6f944067bd205839/crates/swc/tests/fixture/issues-4xxx/4953/input/.swcrc#L19